src/clj/rems/logging.clj

```diff
@@ -0,0 +1,58 @@
+(ns rems.logging
+  (:require [clojure.test :refer [deftest is testing]])
+  (:import (org.slf4j MDC)))
+
+(defn with-mdc* [m f]
```

Comment:
I think this `ns` or this function could use more documentation.

Reply:
There's thorough documentation on lines 27-58; the tests.

Reply:
It would be good to document the namespace still about the intention of the file.